pkg/skaffold/deploy/label/labels.go

```diff
@@ -24,30 +24,26 @@ import (
 	"github.com/sirupsen/logrus"
 	"k8s.io/apimachinery/pkg/api/meta"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
-	"k8s.io/apimachinery/pkg/runtime"
 	"k8s.io/apimachinery/pkg/runtime/schema"
 	"k8s.io/apimachinery/pkg/types"
 	patch "k8s.io/apimachinery/pkg/util/strategicpatch"
 	"k8s.io/client-go/discovery"
 	"k8s.io/client-go/dynamic"
 
+	deploy "github.com/GoogleContainerTools/skaffold/pkg/skaffold/deploy/types"
 	kubernetesclient "github.com/GoogleContainerTools/skaffold/pkg/skaffold/kubernetes/client"
 	kubectx "github.com/GoogleContainerTools/skaffold/pkg/skaffold/kubernetes/context"
 )
 
-// Artifact contains all information about a completed deployment
-type Artifact struct {
-	Obj       runtime.Object
-	Namespace string
-}
-
 // retry 3 times to give the object time to propagate to the API server
 const (
 	tries     = 3
 	sleeptime = 300 * time.Millisecond
 )
 
-func labelDeployResults(labels map[string]string, results []Artifact) error {
+// LabelDeployResults applies all provided labels to the created Kubernetes resources
+//nolint:golint
```

Comment:
```suggestion
```
can we fix the lint offender instead by naming the function something like `Apply` instead of `LabelDeployResults `

Reply:
👍